srcpkgs/kcm-wacomtablet/template

```diff
@@ -14,3 +14,4 @@ license="GPL-2.0-or-later"
 homepage="https://invent.kde.org/system/wacomtablet"
 distfiles="${KDE_SITE}/wacomtablet/${version}/wacomtablet-${version}.tar.xz"
 checksum=c80ce63a41f6fcbb50ac4c2130ed2f8273c4b744e62e33d4b714bf83e8e5f7a4
+make_check=no # FIXME
```

Comment:
Done, but what is it for?

Reply:
> added `libinput-devel` to `makedepends`, but what is it for,

Try clean everything in `hostdir/binpkgs`, cross-build `kcm-wacomtablet` ;)

> and why are you adding it here on a separate line?

Because I can't suggest in the `makedepends` line.